data_warehouse/views.py

```diff
@@ -28,3 +37,33 @@ def get_canvas_options(self, request):
         '''getting canvas-options'''
         data = CanvasOptionsService.get_canvas_options()
         return Response(data)
+
+    @action(detail=False, methods=['GET'], url_path='table-export',
+            url_name='table-export')
+    def export(self, request):
```

Comment:
The view doesn't know your context about table export, I think the validation should be done in your service, and despite you have similar logic with `AggregateDataService.dispatch`, you might need to use another method or service to dispatch your request, and in that dispatch function, validate the parameters. Besides, multiple equality comparisons is bad, consider using:
```python
handlers = {
    TABLE_NAME_EMPLOYEES: xxxx_function,
    ...
}
handler = handlers.get(table_type, None)
if handle is None:
    raise BadRequest
```